flatpak/io.github.celluloid_player.Celluloid.json

```diff
@@ -1,164 +1,403 @@
-                }
-            ]
-        },
-        {
-            "name": "uchardet",
-            "buildsystem": "cmake-ninja",
-            "config-opts": [
-                "-DCMAKE_BUILD_TYPE=Release",
-                "-DBUILD_STATIC=0"
-            ],
-            "cleanup": [
-                "/bin",
-                "/include",
-                "/lib/pkgconfig",
-                "/share/man"
-            ],
-            "sources": [
-                {
-                    "type": "archive",
-                    "url": "https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.6.tar.xz",
-                    "sha256": "8351328cdfbcb2432e63938721dd781eb8c11ebc56e3a89d0f84576b96002c61"
-                }
-            ]
-        },
-        {
-            "name": "libmpv",
-            "cleanup": [ "/include", "/lib/pkgconfig", "/share/man" ],
-            "buildsystem": "simple",
-            "build-commands": [
-              "python3 waf configure --prefix=/app --enable-libmpv-shared --disable-cplayer --disable-build-date --disable-alsa",
-              "python3 waf build",
-              "python3 waf install"
-            ],
-            "sources": [
-              {
-                "type": "git",
-                "url": "https://github.com/mpv-player/mpv.git"
-              },
-              {
-                "type": "file",
-                "url": "https://waf.io/waf-2.0.19",
-                "sha256": "ba63c90a865a9bcf46926c4e6776f9a3f73d29f33d49b7f61f96bc37b7397cef",
-                "dest-filename": "waf"
-              }
-            ]
-        },
-        {
-          "name": "youtube-dl",
-          "no-autogen": true,
-          "no-make-install": true,
-          "make-args": ["youtube-dl", "PYTHON=/usr/bin/python3"],
-          "post-install": ["install youtube-dl /app/bin"],
-          "sources": [{
-            "type": "git",
-            "url": "https://github.com/rg3/youtube-dl.git",
-            "tag": "2020.05.29",
-            "commit": "228c1d685bb6d35b2c1a73e1adbc085c76da6b75"
-          }]
-        },
-        {
-            "name": "celluloid",
-            "buildsystem": "meson",
-            "cleanup": ["/share/man"],
-            "sources": [
-              {
-                "type": "git",
-                "url": "https://github.com/celluloid-player/celluloid.git"
-              }
-            ]
-        }
-    ]
+   "app-id":"io.github.celluloid_player.Celluloid",
+   "runtime":"org.gnome.Platform",
+   "runtime-version":"master",
+   "sdk":"org.gnome.Sdk",
+   "command":"celluloid",
+   "finish-args":[
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=all",
+      "--share=network",
+      "--socket=pulseaudio",
+      "--filesystem=xdg-pictures",
+      "--talk-name=org.gtk.vfs",
+      "--talk-name=org.gtk.vfs.*",
+      "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
```

Comment:
i don't think this is necessary

Reply:
You're right. I removed it in 2801299d411561f0b95c609bb1282497158f16a5.